hathor/manager.py

```diff
@@ -497,15 +499,14 @@ def _initialize_components(self) -> None:
             self.step_validations(depended_final_txs)
             self.log.debug('pending validations finished')
 
-        best_height = self.tx_storage.get_height_best_block()
-        if best_height != h:
-            self.log.warn('best height doesn\'t match', best_height=best_height, max_height=h)
+        # best_height = self.tx_storage.get_height_best_block()
+        # if best_height != h:
+        #     self.log.warn('best height doesn\'t match', best_height=best_height, max_height=h)
```

Comment:
Why have you removed this check?

Reply:
Because there's no `h` variable anymore now, since the loop is inside the indexes and theoretically might not always run (in practice it will because there still are memory indexes that need to be initialized).

Reply:
I feel you fixed it in the last refactor of the initialization, right?

Reply:
Yes.